.github/workflows/docs.yaml

```
@@ -0,0 +1,60 @@
+on:
+  push:
+    branches:
+      - master
```

```suggestion
      - main
```

BTW, is it intentional to publish the `main` branch contents (not tagged contents) to https://arrow.apache.org/datafusion/ ?

FYI: ADBC publishes multiple versions:

* https://arrow.apache.org/adbc/ : Redirects to the latest release
* https://arrow.apache.org/adbc/0.2.0/ : The latest release
* https://arrow.apache.org/adbc/0.1.0/ : An old release
* https://arrow.apache.org/adbc/main/ : The next release (not released yet)

See also:
* https://github.com/apache/arrow-adbc/blob/main/.github/workflows/nightly-website.yml
* https://github.com/apache/arrow-adbc/blob/main/ci/scripts/website_build.sh

---

@alamb Do you have a preference of this?